api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -336,6 +337,14 @@ def serialize_offer(cls, offer: offers_models.Offer) -> dict:
         distinct = extra_data.get("isbn") or extra_data.get("visa") or str(offer.id)
         distinct += extra_data.get("diffusionVersion", "")
 
+        music_type_label = None
+        music_type = (offer.extraData or {}).get("musicType", "").strip()
+        if music_type:
+            try:
+                music_type_label = MUSIC_TYPES_DICT[int(music_type)]
+            except (ValueError, KeyError, TypeError):
+                logger.warning("bad music type encountered", extra={"music_type": music_type})
```

Comment:
(détail) il serait utile de logger l'identifiant de l'offre pour pouvoir plus facilement la corriger.

```suggestion
                logger.warning("bad music type encountered", extra={"offer": offer.id, "music_type": music_type})
```